arduino/cores/packagemanager/package_manager.go

```diff
@@ -156,32 +158,32 @@ func (pm *PackageManager) ResolveFQBN(fqbn *cores.FQBN) (
 	targetPackage := pm.Packages[fqbn.Package]
 	if targetPackage == nil {
 		return nil, nil, nil, nil, nil,
-			errors.New("unknown package " + fqbn.Package)
+			fmt.Errorf(tr("unknown package %s"), fqbn.Package)
 	}
 
 	// Find platform
 	platform := targetPackage.Platforms[fqbn.PlatformArch]
 	if platform == nil {
 		return targetPackage, nil, nil, nil, nil,
-			fmt.Errorf("unknown platform %s:%s", targetPackage, fqbn.PlatformArch)
+			fmt.Errorf(tr("unknown platform %s:%s"), targetPackage, fqbn.PlatformArch)
```

Comment:
Why not use the `%[1]s:%[2]s` syntax here like in other places? 🤔

Reply:
Ok, makes sense. I didn't understand the need of the index syntax I guess. 👍

Reply:
In a case where it was something like "unknown architecture %s for vendor %s", the index syntax is important because in some languages, the natural sentence structure might dictate that the two printf verbs be placed in the opposite order from how they happen to occur in the English source sentence. This is impossible to achieve without the indices and the translator should not be expected to have the in depth understanding of Go to know how to add the indexes as needed.

I suppose an argument could be made that the indexes should be added everywhere for the sake of consistency, but that also might send the wrong message to the translator. From my involvement in the Arduino language reference translation efforts, and even the classic Arduino IDE translations, I have observed that inappropriate translation of code content is a common problem.